pytorch_lightning/trainer/callback_config.py

```diff
@@ -62,7 +62,7 @@ def configure_checkpoint_callback(self):
             self.weights_save_path = self.default_save_path
 
     def configure_early_stopping(self, early_stop_callback):
-        if early_stop_callback is True:
+        if early_stop_callback is True or None:
```

Comment:
I do not think that this forsing default early stop unles you explicitly say no... is good